sphinx-docs/index.rst

```diff
@@ -9,7 +9,7 @@ KA Lite Documentation
 Welcome to the KA Lite Documentation page! Here, you will find all the information needed to set up the KA Lite software. Additionally, there's information on how to manage KA Lite and how to perform role-specific tasks. If you have any questions that are not addressed by the documentation and you have access to the Internet, please be sure to check our `FAQ`_ or `Forums`_!    
 
 
-.. _FAQ: https://learningequality.org/ka-lite/faq/
+.. _FAQ: https://learningequality.org/docs/faq.html
```

Comment:
You can also [cross reference](http://sphinx-doc.org/markup/inline.html#cross-referencing-documents) this document, instead of linking to it directly.